assets/scss/components/place.scss

```diff
@@ -0,0 +1,137 @@
+.main-content-place {
+    height: 88%;
+
+    .place-info {
+        padding: 2em 7em;
+        height: 85%;
+    
+        .place-grid {
+            display: grid;
+            grid-template-columns: 30% 70%;
+            grid-template-rows: 75% 25%;
+           
+            height: calc(100% - 7em);
+           
+            .place-grid__item {
+                
+                &:nth-of-type(1) {
+                    grid-column: 1 / 2;
+                    grid-row: 1 / 2;
+                
+                    img {
+                        height: 100%;
+                        width: 100%;
+                        border-radius: 50%;
+                    }
+                }
+                
+                &:nth-of-type(2) {
+                    grid-column: 2 / 3;
+                    grid-row: 1 / 3;
+                    
+                    margin-left: 2em;
+                    border: 2px black solid;
+                    
+                    display: flex;
+                    justify-content: center;
+                    flex-direction: column;
+                    
+                    .place-info__title {
+                        padding:0.3em;
+
+                        height: 20%;
+                        font-size: 2.5em;
+                        text-align: center;
+                    }
+                    
+                    .place-info__container {
+                        width: 100%;
+                        height: 100%;
+                    
+                        display: flex;
+                        justify-content: center;
+                    
+                        .place-info__colaborator-container {
+                            width: 70%;
+                            position: relative;
+
+                            border-top: 2px black solid;
+
+                            .place-info__colaborator {
+                                position: absolute;
+                                top:10%;
+                                left:50%;
+                                transform: translate(-50%,-10%);
```

Comment:
`transform: translate(-50%, -50%)` are rolul de a alinia centrul unui element cu centrul elementului părinte.  Acel `-10%` pe axa Y nu are sens pentru mine.

Reply:
Poate am înteles eu greșit, dar dacă bag transformu ală nu mă ajută să ma aliniez cu părintele? Adica nu functionează pe acelas principiu ca la 50%? Gen sa fie fix acolo, nu sa fie oleaca mai incolo cum e la 50%.

Reply:
- Dacă aliniezi ceva cu `position: absolute` **fără** `transform`: 
  - se aliniază colțul din stânga sus a elementului cu `position: absolute` la coordonatele pe care le setezi cu `top`, `right`, `bottom`, `left`.
- Dacă pe lângă `position: absolute` **folosești** `transform`:
  - pentru valoarea `translate3d(-50%, -50%, 0)` se aliniază fix centrul elementului cu `position: absolute` 
P.S. : procentajele din `translate` se calculează pe baza dimensiunii elementului pe care aplici `transform`-ul.

În cazul tău acel `10%` de la `top`  nu reprezintă aceeași valoare ca acel `10%` din `translate`.